index.js

```diff
@@ -980,6 +980,48 @@
   //. ```
   S.init = S.slice(0, -1);
 
+  //# take :: Number -> [a] -> Maybe [a]
+  //.
+  //. Returns Just the first N elements of the given collection if N is
+  //. greater than or equal to zero and less than or equal to the length
+  //. of the collection; Nothing otherwise. Supports Array, String, and
+  //. any other collection type which provides a slice method.
+  //.
+  //. ```javascript
+  //. > S.take(2, ['a', 'b', 'c', 'd', 'e'])
+  //. Just(["a", "b"])
+  //.
+  //. > S.take(4, 'abcdefg')
+  //. Just("abcd")
+  //.
+  //. > S.take(4, ['a', 'b', 'c'])
+  //. Nothing()
+  //. ```
+  S.take = R.curry(function(n, xs) {
+    return n < 0 ? Nothing() : slice(0, n, xs);
+  });
+
+  //# drop :: Number -> [a] -> Maybe [a]
+  //.
+  //. Returns Just all but the first N elements of the given collection if N is
+  //. greater than or equal to zero and less than or equal to the length of the
+  //. collection; Nothing otherwise. Supports Array, String, and any other
+  //. collection type which provides a slice method.
```

Comment:
This is *very* minor, but we could tweak the line wrapping to increase the aesthetic appeal of this paragraph:

```javascript
  //. Returns Just all but the first N elements of the given collection
  //. if N is greater than or equal to zero and less than or equal to the
  //. length of the collection; Nothing otherwise. Supports Array, String,
  //. and any other collection type which provides a `slice` method.
```

One day I'll write a function to describe the attractiveness of paragraphs in text files. :)